src/board/v3/error.c

```diff
@@ -71,7 +81,7 @@ void _screaming_death(const char *pos, const char *msg)
   sbp_state_init(&sbp_state);
 
   /* Continuously send error message */
-  #define APPROX_ONE_SEC 33000000
+  #define APPROX_ONE_SEC 200000000
```

Comment:
I forget the exact reason, but we're actually using one of the timer peripherals as the cycle counter for ChibiOS on v3. So for purposes of being failsafe, I think this should stay the way it is (though I agree it's terrible).